BasiliskII/src/MacOSX/clip_macosx64.mm

```diff
@@ -1124,7 +1126,7 @@ static void ConvertHostPasteboardToMacScrap()
 	NSData *textData = MacTEXTAndStylDataFromPasteboard(g_pboard, &stylData);
 
 	if (textData) {
-		if (stylData)
+		if (stylData && [stylData length] > 2)
```

Comment:
Can you add a comment about the length check? It's not clear why it's needed.

Reply:
Honestly, I don't know anything under Mac OS X folders. XCode build didn't work on mine, either.

But in Mac OS X autoconf build requires some files under MacOSX folder.